gui/wxpython/startup/guiutils.py

```diff
@@ -320,6 +321,170 @@ def delete_mapset_interactively(guiparent, grassdb, location, mapset):
     dlg.Destroy()
 
 
+def delete_mapsets_interactively(guiparent, grassdbs, locations, mapsets):
```

Comment:
this shouldn't take as input nodes, it should be something like a list of tuples [(db1, loc1, mapset1), (), ...]